src/pages/jp.md

```diff
@@ -0,0 +1,141 @@
+---
+title: コンセントについて
+---
+import { Image, License } from '../theme/consento'
+
+--- 
+
+## 人間による、人間のためのデータプライバシー。
+
+Consentoは、あなたの組織のすべての人にHuman-Factor Authentication（人介認証）を提供します。パスワードレス。クラウドフリー。協調性があります。あなたの組織が安心して機密業務へのアクセスと共有を管理するために必要なものです。
+
+---
+
+## 課題
+
+デジタルトランスフォーメーションがもたらす多くの可能性は、データプライバシーに関する課題を伴います。これらの課題は、技術的なシステムよりも、機密データをまとめて管理することのストレスにあります。Consentoは、プロフェッショナルや中小企業が、分散型で、パスワード不要の多要素認証ハブを独自で構築できるようにします。私たちのアプリは、あなたのデバイスを鍵に変えることで機密データへのアクセスが一括して保護されるようになります。機密データへのアクセスを管理する責任が共有されます。
+
+---
+
+## 使用例
+### クリプトサービスのユーザー
+> 単一のユーザーのプライベートデータを保管するユースケース。
+
+ビットコインや分散型台帳技術は、オンラインでの安全な取引に素晴らしい機会をもたらします。しかし、その技術性の高さから、ほとんどのエンドユーザーは少し不安を感じています。さらに、エンドユーザーを「単一障害点」に変えてしまうことになります。
+
+Consentoでは、アクセストークンをユニークなパーツに分割して、あなたが選んだデバイス（あなたとあなたが信頼する人のデバイス）に安全に保管します。認証のために、Consentoはあなたのリクエストに応じてパーツを集めます。パスワードを覚える必要はありません。
+
+### リモートワークのユーザー
+> チームデータの保管責任共有のユースケース。
+
+リモートワークは、24時間365日、仕事とプライベートが混在しているという不便さに加え、リモートワーカーや、会社のデータプライバシーを管理する担当者にとって、別のストレスの原因となります。リモートワーク環境下では、データ漏洩やサイバーセキュリティの脅威がこれまでになく高まっています。
+
+Consentoでは、同僚の助けを借りて、機密データや会社のサービスに安全にアクセスすることができます。会社のセキュリティで保護されたネットワークの外で行われる作業であっても、会社の機密を扱う責任は分散され、一人の人間が誤操作することによるストレスは軽減されます。また、人事部やIT部門は、取引先やビジネスパートナーはもちろんのこと、すべての従業員を簡単に登録することができます。
+
+### ソフトウェアエスクローユーザー
+> 複数のステークホルダーによるデータ保管のユースケース。
+
+ソフトウェアエスクローサービスは、開発会社が倒産した際、企業にコードのライセンス使用を保証するサービスです。ソフトウェアエスクローは、開発会社が倒産しても、開発者のコードがずっと管理されていることを保証します。このようなシステムでは、様々な数のパーシャルキーが必要となり、自動化やコードの検証が必要となります。
+
+Consentoでは、エスクローシステムが作成され、異なるパーティに分割されます。パーティの数は調整可能です。データは異なるデバイス間で同期が保たれ、データはいつでも検証することができます。
+
+---
+
+## ソリューション
+Consentoのアプリは、あなたの組織にカスタムメイドのHuman-Factor Authenticationを構築することを可能にします。
+ 
+<figure className="kg-card kg-image-card kg-card-hascaption">
+  <img src="/img/tanja/cc-by-nc-sa/icons/passwordless.svg" style={{ float: 'left', width: '20%' }} />
+<figcaption><strong>パスワードレス</strong>なので、ハッキングの対象にならず、パスワードの管理ミスのリスクもありません。</figcaption>
+</figure>
+<br/>
+
+---
+
+<br/>
+<figure className="kg-card kg-image-card kg-card-hascaption">
+  <img src="/img/tanja/cc-by-nc-sa/icons/icon_lock.svg" style={{ float: 'left', width: '20%' }} />
+<figcaption><strong>クラウドフリー</strong>なので、中央管理者を必要とせず、認証プロセスはユーザーだけのものとなります。あなたのデータ管理はあなたの財産です。</figcaption>
+</figure>
+<br/>
+
+---
+
+<br/>
+<figure className="kg-card kg-image-card kg-card-hascaption">
+  <img src="/img/tanja/cc-by-nc-sa/icons/collaborative.svg" style={{ float: 'left', width: '20%' }} />
+<figcaption><strong>相互の応答性</strong>Consentoは、組織内やサプライヤー、顧客に配布される部分的な鍵によって、認証をより安全にします。一人のユーザーがシステム全体の故障の原因になることはありません。</figcaption>
+</figure>
+<br/>
+
+---
+
+<br/>
+<figure className="kg-card kg-image-card kg-card-hascaption">
+  <img src="/img/tanja/cc-by-nc-sa/icons/icon_lock.svg" style={{ float: 'left', width: '20%' }} />
```

Comment:
This is the same icon as two sections up.

Reply:
I update the file, and replaced the repeated image